src/chart/Pie.js

```diff
@@ -32,6 +32,9 @@
     Pie.prototype.implements(I2DChart.prototype);
 
     Pie.prototype.publish("paletteID", "default", "set", "Palette ID", Pie.prototype._palette.switch(),{tags:['Basic','Shared']});
+    Pie.prototype.publish("outerText", false, "boolean", "Sets label position inside or outside chart",null,{tags:['Basic']});
+    Pie.prototype.publish("radius", 100, "number", "Sets radius of pie chart",null,{tags:['Basic']});
+    Pie.prototype.publish("innerRadius", 0, "number", "Sets radius of center hole in pie chart (donut)",null,{tags:['Basic']});
```

Comment:
Once properties are published the "overriding" methods need to call to "new" corresponding base class methods